app/assets/javascripts/admin/addon/templates/web-hooks-show.hbs

```diff
@@ -1,92 +1,43 @@
-<LinkTo @route="adminWebHooks" class="go-back">
-  {{d-icon "arrow-left"}}
-  {{i18n "admin.web_hooks.go_back"}}
-</LinkTo>
+<div class="web-hook-direction">
```

Comment:
I pulled this branch locally, and the events page could use a title. Right now you can't tell you're inside of an individual webhook's events page: 

<img width="1105" alt="image" src="https://user-images.githubusercontent.com/368961/206348446-cab00e7c-38be-446a-af33-fb3e14fab7b0.png">

There isn't a webhook name, but maybe we can use a part of the URL or its ID... or something.